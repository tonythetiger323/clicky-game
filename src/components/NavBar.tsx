import React from "react";
import AppBar from "@material-ui/core/AppBar";
import Toolbar from "@material-ui/core/Toolbar";
import Typography from "@material-ui/core/Typography";

export default function NavBar() {
  return (
    <AppBar position="static">
      <Toolbar>
        <Typography variant="h6">Clicky Game</Typography>
        <Typography variant="h6">Click an image to begin!</Typography>
        <Typography variant="h6">Score: 0 | Top Score: 0</Typography>
      </Toolbar>
    </AppBar>
  );
}
