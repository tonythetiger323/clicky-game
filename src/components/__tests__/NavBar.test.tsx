import React from "react";
import { shallow } from "enzyme";
import NavBar from "../NavBar";

describe("App", () => {
  const navbar = shallow(<NavBar />);

  it("renders correctly", () => {
    expect(navbar).toMatchSnapshot();
  });
});
