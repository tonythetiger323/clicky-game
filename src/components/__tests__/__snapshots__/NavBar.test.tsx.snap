// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<WithStyles(ForwardRef(AppBar))
  position="static"
>
  <WithStyles(ForwardRef(Toolbar))>
    <WithStyles(ForwardRef(Typography))
      variant="h6"
    >
      Clicky Game
    </WithStyles(ForwardRef(Typography))>
    <WithStyles(ForwardRef(Typography))
      variant="h6"
    >
      Click an image to begin!
    </WithStyles(ForwardRef(Typography))>
    <WithStyles(ForwardRef(Typography))
      variant="h6"
    >
      Score: 0 | Top Score: 0
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Toolbar))>
</WithStyles(ForwardRef(AppBar))>
`;
